experimental/dds/tree/src/EditLog.ts

```diff
@@ -7,7 +7,8 @@ import BTree from 'sorted-btree';
 import { TypedEventEmitter, assert } from '@fluidframework/common-utils';
 import type { IEvent } from '@fluidframework/common-definitions';
 import { ITelemetryLoggerExt } from '@fluidframework/telemetry-utils';
-import { compareArrays } from '@fluidframework/core-utils';
+// eslint-disable-next-line import/no-internal-modules
+import { compareArrays } from '../../../../packages/common/core-utils';
```

Comment:
TS doesn't rewrite imports. If we want to do something like this, we'd need a strategy to make an import statement like this resolve correctly for any consumer of the shared-tree package. That makes relative imports to files outside of the package scope somewhat infeasible.